aws/table_aws_wafv2_web_acl.go

```diff
@@ -98,6 +98,34 @@ func tableAwsWafv2WebAcl(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_BOOL,
 				Hydrate:     getAwsWafv2WebAcl,
 			},
+			{
+				Name:        "custom_response_bodies",
+				Description: "The custom response bodies that AWS WAF returns to web requests when they are blocked.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsWafv2WebAcl,
+				Transform:   transform.FromField("CustomResponseBodies"),
```

Comment:
Is the explicit transform necessary here, or would the default transform handle this correctly?